src/BackgroundImage.js

```diff
@@ -1,9 +1,21 @@
 import styled from 'styled-components'
 import PropTypes from 'prop-types'
 
-const image = props => props.image
-  ? { backgroundImage: `url(${props.image})` }
-  : null
+import {
+  detectWebPSupport
+} from './helpers'
+
+const image = props => {
+  /* istanbul ignore next */
+  const isWebP = props.useWebP && detectWebPSupport()
+  /* istanbul ignore next */
```

Comment:
These Istanbul ignore statements support the bypassing of the WebP Detection in Instanbul test coverage.